pyiso/caiso.py

```diff
@@ -283,7 +284,24 @@ def get_AS_dataframe(self, anc_region, latest=True, start_at=False, end_at=False
         MW columns holds $/MW float.
         If no data, returns an empty dataframe.
         """
-        # set args
+        # set args, handle latest differently than LMP b/c daily publication schedule
```

Comment:
Could this be made a separate method with its own tests? That would help me understand what edge cases it's intending to handle. Also, what if we make latest=True return today always, and tomorrow=True return tomorrow if it's available yet... would that work for the downstream problem you're addressing here?